### DISCUSSION TITLE
Instructions unclear for copying a local git repository on Linux, on to GitHub.

### DISCUSSION THREAD

**Discussion by TimeLikeAlpha** [community/community#34614] [New to GitHub]
Adding a local repository to GitHub using Git

1) I go to page https://docs.github.com/en/get-started/importing-your-projects-to-github/importing-source-code-to-github/adding-locally-hosted-code-to-github , then (after 15 minutes) find the little "Linux" tab, and get to Step 7.  **Step 7 features a little icon to click that is not visible on the page in question.  I do have a different icon, of two superimposed squares, but I don't know either of these icons.   I think this icon is just to select text?**  Since this is for expert users, why not simply say "select the URL" or something?

2) After I create the repository it then gives me somewhat different instructions when I follow them I get the error 

```
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on

```

**Why are you giving instructions two different places for something you don't seem to even support any more?**

3) I then try git remote add origin MY_URL and get the error

`fatal: remote origin already exists.
`
Why so?

**Comment by airtower-luna** [comment]
> I do have a different icon, of two superimposed squares, but I don't know either of these icons. I think this icon is just to select text?

The icon seems to have changed since the docs were made, but that button should *copy* the URL. If you copy the URL by other means it'll work just as well. :slightly_smiling_face: 

For your second point, do as the linked docs say and [use a PAT](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) instead of your password. You need at least the **repo** scope, and **workflow** if you use actions.

> I then try git remote add origin MY_URL and get the error
> 
> `fatal: remote origin already exists.`
> Why so?

Depends on what exactly you did before, evidently you already created a remote called `origin`.

And please don't yell at us other users if you don't like GitHub company decisions. :wink:
